daemon/src/maker_inc_connections.rs

```diff
@@ -236,12 +236,14 @@ impl Actor {
             .get_mut(taker_id)
             .ok_or(NoConnection(*taker_id))?;
 
-        if conn.send(msg).await.is_err() {
-            self.drop_taker_connection(taker_id).await;
-            return Err(NoConnection(*taker_id));
+        match conn.send(msg).await {
+            Ok(_) => Ok(()),
```

Comment:
Is the ignored value here a `()`? Would be good to explicitly match against so it becomes a compile-error in case anyone ever changes the signature of this function.

Reply:
done